### DISCUSSION TITLE
Does Jack like towns or villages?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1117] [Galaxy Trivia] [answered]
He likes balloons, but does not like parties!
He likes books, but does not like reading!
He likes weeds, but does not like flowers!
He likes swimming, but does not like water!

**Comment by peterbaccaro** [comment]
Jack like **Villages** because he only likes words that contain double letter.

**Comment by thuongtruong1009** [comment, marked as answer]
it seems like Jack prefers towns over villages. The pattern in the clues suggests that Jack likes things that are associated with the word but not the actual activity or object. For instance:

He likes balloons, but does not like parties: Jack enjoys the word "balloons" but not the word "parties."
He likes books, but does not like reading: Jack is fond of the word "books" but not the activity of "reading."
He likes weeds, but does not like flowers: Jack has a preference for the word "weeds" but not the word "flowers."
He likes swimming, but does not like water: Jack is interested in the concept of "swimming" but not in the substance of "water."
Following this pattern, it can be inferred that Jack likes the idea of "towns" but not "villages."
